test/e2e_flags.go

```diff
@@ -52,6 +64,11 @@ func (ps *Provisioners) String() string {
 func (ps *Provisioners) Set(value string) error {
 	for _, provisioner := range strings.Split(value, ",") {
 		provisioner := strings.TrimSpace(provisioner)
+		if !isValid(provisioner) {
+			fmt.Printf("The given provisioner %q is not supported, tests cannot be run.\n", provisioner)
```

Comment:
Prefer `Fatalf` here and everywhere, so the error will be properly shown in the test summary.

Reply:
Since `fmt` does not have `Fatalf` method, I've used `Printf` and `os.Exit(1)` instead.

Reply:
Use `log` or `logger` then, whatever is more appropriate here.

Reply:
Done.